contracts/TokenHandler.sol

```diff
@@ -138,6 +137,8 @@ contract TokenHandler is ITokenHandler, ITokenManagerType, ReentrancyGuard, Crea
         // For lock/unlock token managers, the ITS contract needs an approval from the token manager to transfer tokens on its behalf
```

Comment:
maybe change `address tokenManager` -> `ITokenManager tokenManager` to make it cleaner.

Since `tokenHandler` bytecode doesn't need to be fixed, could make `tokenManagerType` typed as well